### DISCUSSION TITLE
Is there a way to change the position of the error text in TextInput ?

### DISCUSSION THREAD

**Discussion by sahrohit** [mantinedev/mantine#989] [Q&A] [answered]
The error message is at the bottom of the input field, and whenever there is an error it pushes the other fields below it. So usually I keep an error message with the label ?

Is it possible to achieve this?

**Comment by rtivital** [comment, marked as answer]
You can customize input elements with Styles API

**Reply by rtivital** [reply]
https://codesandbox.io/s/quirky-antonelli-xqwn3u
